Remotion/Web/Core/ExecutionEngine/WxePageInfo.cs

```diff
@@ -372,12 +371,13 @@ private void RegisterWxeInitializationScript ()
       StringBuilder initScript = new StringBuilder (500);
 
       initScript.AppendLine ("WxePage_Context.SetInstance (new WxePage_Context (");
-      initScript.AppendLine ("    ").Append (isCacheDetectionEnabled).AppendLine (",");
-      initScript.AppendLine ("    ").Append (refreshInterval).AppendLine (",");
-      initScript.AppendLine ("    ").Append (refreshPath).AppendLine (",");
-      initScript.AppendLine ("    ").Append (abortPath).AppendLine (",");
-      initScript.AppendLine ("    ").Append (statusIsAbortingMessage).AppendLine (",");
-      initScript.AppendLine ("    ").Append (statusIsCachedMessage).AppendLine ("));");
+      initScript.Append ("    ").Append (isCacheDetectionEnabled).AppendLine (",");
+      initScript.Append ("    ").Append (refreshInterval).AppendLine (",");
+      initScript.Append ("    ").Append (refreshPath).AppendLine (",");
+      initScript.Append ("    ").Append (abortPath).AppendLine (",");
+      initScript.Append ("    ").Append (statusIsAbortingMessage).AppendLine (",");
+      initScript.Append ("    ").Append (statusIsCachedMessage).AppendLine (",");
+      initScript.Append ("    ").Append (wxeContext.PostBackID + 1).AppendLine ("));");

```

Comment:
In line 369 (after isCacheDeectionEnabled = ...), add the `int nextPostBackID = wxeContext.PostBackID + 1;` and use the nextPostBackID here, please, to remove the logic from the rendering statement.